cli/src/config/templates.toml

```diff
@@ -156,12 +140,26 @@ commit_summary_separator = 'label("separator", " | ")'
   time_range.start().ago() ++ label("time", ", lasted ") ++ time_range.duration()'''
 'format_timestamp(timestamp)' = 'timestamp.local().format("%Y-%m-%d %H:%M:%S")'
 
+format_op_log_operation = '''
+  label(if(current_operation, "current_operation"),
+    concat(
+      separate(" ", id.short(), user, format_time_range(time)), "\n",
+      description.first_line(), "\n",
+      if(tags, tags ++ "\n"),
+    ),
+  )
+'''
+format_op_log_snapshot = 'format_op_log_operation'
+# the 'builtin' version is deprecated
+'builtin_op_log_root(op_id)' = 'separate(" ", op_id.short(), label("root", "root()")) ++ "\n"'
+format_op_log_root = 'builtin_op_log_root(id)'
```

Comment:
ditto, `format_op_log_root(op)` though `op` should be the root operation.

Reply:
The thing is (was with the root commit and then was copied to the root op) that those methods are generic in case there's a potential backend which has a different id for the root commit/op ¯\\\_(ツ)_/¯